assert/assert.go

```diff
@@ -128,9 +128,9 @@ func TempDir(tb testing.TB) (dir string, cleanup func()) {
 	}
 }
 
-func WriteFile(tb testing.TB, fp, data []byte, perms os.FileMode) () {
+func WriteFile(tb testing.TB, fp, data []byte, perms os.FileMode) {
 	tb.Helper()
 
-	err := os.WriteFile(fp, data, perms)
+	err := os.WriteFile(string(fp), data, perms)
```

Comment:
Oh nah can you not make this change, I've got a PR inc. just ignore the CI warning and I'll force merge.

Reply:
Oh actually I guess you'll have to. Ok no big deal make `fp` of type `string` here.

Reply:
toc is also failing on master

Reply:
I just pushed the fix for WriteFile to master directly.

Reply:
Nvm I see it now.